runtime/lua/vim/fs.lua

```diff
@@ -86,6 +86,64 @@ function M.basename(file)
   return file:match('/$') and '' or (file:match('[^/]*$'))
 end
 
+--- @class vim.fs.abspath.Opts
+--- @inlinedoc
+---
+--- Current working directory to use as the base for the relative path.
+--- If omitted, the |current-directory| is used.
+--- This option is also ignored for C:foo\bar style paths on Windows, as those paths use the current
+--- directory of the drive specified in the path.
+--- @field cwd? string
```

Comment:
```suggestion
--- Current working directory to use as the base for the relative path.
--- This option is also ignored for C:foo\bar style paths on Windows, as those paths use the current
--- directory of the drive specified in the path.
--- (default: |current-directory|)
--- @field cwd? string
```

Should produce nicer docs.